tests/samples/CMakeLists.txt

```diff
@@ -9,8 +9,7 @@ iree_lit_test_suite(
     lit
   SRCS
     "pack_pipeline_funcIR.mlir"
-    "pack_pipeline_funcIR_2core.mlir"
-    "pack_pipeline_funcIR_4core.mlir"
+    "pack_pipeline_funcIR_4core_256_256_256.mlir"
```

Comment:
I think you can remove this test and then this PR is ready to merge.

Reply:
Done